ompi/communicator/ft/comm_ft.c

```diff
@@ -154,7 +162,9 @@ int ompi_comm_shrink_internal(ompi_communicator_t* comm, ompi_communicator_t** n
                                         comm,
                                         comm->c_coll->coll_agree_module);
     } while( MPI_ERR_PROC_FAILED == ret );
+#if OPAL_ENABLE_DEBUG
     stop = PMPI_Wtime();
+#endif
     OPAL_OUTPUT_VERBOSE((10, ompi_ftmpi_output_handle,
                          "%s ompi: comm_shrink: AGREE: %g seconds",
                          OMPI_NAME_PRINT(OMPI_PROC_MY_NAME), stop-start));
```

Comment:
I think we also need `#if OPAL_ENABLE_DEBUG` guards around the `OPAL_OUTPUT_VERBSE()` call which prints `stop-start`?

Reply:
All caps OPAL_OUTPUT_VERBOSE is the version that only expands to something with --enable-debug.  That's why there was a warning in the first place.

Reply:
AH I see.  So when `OPAL_ENABLE_DEBUG` isn't set, it expands to nothing, meaning no usage of `stop-start`!  Thanks.